app/packages/core/src/components/Actions/similar/Helper.tsx

```diff
@@ -1,29 +1,34 @@
-import React from "react";
 import { PopoutSectionTitle, useTheme } from "@fiftyone/components";
+import React from "react";
+
 import { SORT_BY_SIMILARITY } from "../../../utils/links";
 import { ActionOption } from "../Common";
+import useSimilarityType from "@fiftyone/state/src/hooks/similaritySearch/useSimilarityType";
```

Comment:
Can this import avoid accessing `src`?
```suggestion
import { useSimilarityType } from "@fiftyone/state";
```